castle/cms/authentication.py

```diff
@@ -89,6 +89,19 @@ def two_factor_enabled(self):
             enabled = self.registry.get('plone.two_factor_enabled', False)
         return enabled
 
+    @property
+    def request_access(self):
+        if not self.is_zope_root and self.registry:
+            return self.registry.get('plone.request_access_enabled', False)
+        return False
+
+    @property
+    def request_form_url(self):
+        url = '/@@request-form'
+        if not self.is_zope_root and self.registry:
+            url = self.registry.get('plone.request_form_url', '/@@request-form')
```

Comment:
is this plone.request_form_url anymore, or should this be plone.request_access_form_path now?